src/components/layout/Layout.tsx

```diff
@@ -96,18 +97,21 @@ export type LayoutProps = {
 };
 
 export const Layout = ({
+  className,
   content,
   footer,
   headerContent,
   logoSidebar,
   searchInput,
   sidebar,
 }: LayoutProps) => (
-  <AminoLayout>
-    {!!headerContent && <Header>{headerContent}</Header>}
+  <AminoLayout className={className}>
+    {!!headerContent && (
+      <Header className="amino-layout-header">{headerContent}</Header>
+    )}
     <ContentGrid $hasHeader={!!headerContent}>
-      <StyledSidebar>
-        <SidebarContent>
+      <StyledSidebar className="amino-layout-sidebar">
+        <SidebarContent className="amino-layout-sidebar-content">
```

Comment:
are you having those classes styled somewhere?

Reply:
I realized that this Layout component is hard to style with just styled components, so I wanted to make it easier in the future.

Reply:
I'm concerned about classes slowing down future development. This would mean that all styling changes would need to be coordinated with unknown use-cases in other projects.

What changes to the Layout are giving you trouble?